src/common/ceph_context.cc

```diff
@@ -381,17 +381,17 @@ void CephContext::do_command(std::string command, cmdmap_t& cmdmap,
     } else if (command == "config get") {
       std::string var;
       if (!cmd_getval(this, cmdmap, "var", var)) {
-	f->dump_string("error", "syntax error: 'config get <var>'");
+        f->dump_string("error", "syntax error: 'config get <var>'");
```

Comment:
see https://github.com/ceph/ceph/blob/master/CodingStyle#L77. and again, please don't apply formatting only changes to the code you are not changing otherwise.